# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1541081239 14400
#      Thu Nov 01 10:07:19 2018 -0400
# Node ID 97e63ba8a34cb59907f840029f74402b3ce2a8dd
# Parent  f5e08983d07cabbc3ab181becf1d79ef32593b16
* __ode15__.cc (IDA::print_stat): Use octave_stdout instead of std::cout.

diff --git a/libinterp/dldfcn/__ode15__.cc b/libinterp/dldfcn/__ode15__.cc
--- a/libinterp/dldfcn/__ode15__.cc
+++ b/libinterp/dldfcn/__ode15__.cc
@@ -19,29 +19,28 @@ along with Octave; see the file COPYING.
 <https://www.gnu.org/licenses/>.
 
 */
 
 #if defined (HAVE_CONFIG_H)
 #  include "config.h"
 #endif
 
-#include <iostream>
-
 #include "dColVector.h"
 #include "dMatrix.h"
 #include "dSparse.h"
 
 #include "Cell.h"
 #include "defun-dld.h"
 #include "error.h"
 #include "errwarn.h"
 #include "oct-map.h"
 #include "ov.h"
 #include "ovl.h"
+#include "pager.h"
 #include "parse.h"
 
 #if defined (HAVE_SUNDIALS)
 
 #  if defined (HAVE_IDA_IDA_H)
 #    include <ida/ida.h>
 #  endif
 
@@ -887,22 +886,22 @@ namespace octave
       error ("IDA failed to return the number of internal steps");
 
     if (IDAGetNumErrTestFails (mem, &netfails) != 0)
       error ("IDA failed to return the number of internal errors");
 
     if (IDAGetNumResEvals (mem, &nrevals) != 0)
       error ("IDA failed to return the number of residual evaluations");
 
-    std::cout << nsteps << " successful steps\n";
-    std::cout << netfails << " failed attempts\n";
-    std::cout << nrevals << " function evaluations\n";
-    // std::cout << " partial derivatives\n";
-    // std::cout << " LU decompositions\n";
-    // std::cout << " solutions of linear systems\n";
+    octave_stdout << nsteps << " successful steps\n";
+    octave_stdout << netfails << " failed attempts\n";
+    octave_stdout << nrevals << " function evaluations\n";
+    // octave_stdout << " partial derivatives\n";
+    // octave_stdout << " LU decompositions\n";
+    // octave_stdout << " solutions of linear systems\n";
   }
 
   ColumnVector
   ida_user_function (const ColumnVector& x, const ColumnVector& xdot,
                      double t, octave_function *ida_fc)
   {
     octave_value_list tmp;
 
